pkg/certificate/get.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2023 The Bestchains Authors.
```

Comment:
Should we move this unduer `pkg/depository`  as this only works for `depository service`

Reply:
Agree.

Reply:
@dayuy

Reply:
> @dayuy

Any suggestions for command format?
```shell
#? 
bc-cli get depository certificate --kind
```